docs/development/extensions-core/google.md

```diff
@@ -34,6 +34,13 @@ To use this Apache Druid extension, make sure to [include](../../development/ext
 
 To configure connectivity to google cloud, run druid processes with `GOOGLE_APPLICATION_CREDENTIALS=/path/to/service_account_keyfile` in the environment.
 
+|Property|Description|Possible Values|Default|
```

Comment:
Hm, this "Required Configuration" and the "Configuration" section that starts at line 56 should probably be merged, the new wording you have is better so let's use that

Reply:
fixed